osu.Game/Rulesets/Mods/ModMuted.cs

```diff
@@ -89,7 +91,7 @@ public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
             currentCombo = scoreProcessor.Combo.GetBoundCopy();
             currentCombo.BindValueChanged(combo =>
             {
-                double dimFactor = Math.Min(1, (double)combo.NewValue / MuteComboCount.Value);
+                double dimFactor = MuteComboCount.Value == 0 ? 1 : (double)combo.NewValue / MuteComboCount.Value;
 
                 if (InverseMuting.Value)
                     dimFactor = 1 - dimFactor;
```

Comment:
Either this should be bypassed or (more correctly, if feasible) the inverse option should be set to false and disabled when the final volume combo is zero.

Reply:
Disabling it may not work exactly with `Mod.CopyFrom` as it could potentially transfer a value while the bindable is disabled (see https://github.com/ppy/osu-framework/issues/3218).

One other way to go with this however is changing the limits of the mute combo count to minimum 1 on the inverse option, which would also work with `Mod.CopyFrom` given that it does a full bind which transfers the limits as well.